### DISCUSSION TITLE
Is there a way to change the camera position when a button is clicked?

### DISCUSSION THREAD

**Discussion by FehmiCan16** [google/model-viewer#4146] [Q&A]
We can change the camera position by pressing the right mouse button. What I want is to make the camera position go to a certain position when an html element is clicked. How can I do that?

**Comment by elalish** [comment]
I would recommend adding a click event listener to the the HTML element and setting the MV `cameraOrbit` & `cameraTarget` there.

**Reply by FehmiCan16** [reply]
yes i found to change camera target but when i type camare-target="12m 14m 16m" target point goes too far i guess it changes target in meters. When I type this as camera-target="0.016m 0.05m 0.006m" or camera-target="16cm 5cm 6cm" it doesn't work. When I check it from the editor, it gives the target positions as a value between 0 and 1. How can I print the value like this?

**Reply by elalish** [reply]
targets are in meters, and you appear to be specifying units correctly - can you share a link where we can see your repro? The editor isn't actually between 0 and 1, it's just meters.
